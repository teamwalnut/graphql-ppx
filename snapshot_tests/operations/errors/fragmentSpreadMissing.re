[%graphql
  {|
  query MyQuery {
    lists {
      ...MissingFragment
    }
  }
  |}
];
