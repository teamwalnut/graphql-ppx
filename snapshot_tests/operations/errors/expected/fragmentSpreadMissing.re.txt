
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/fragmentSpreadMissing.re[0m:[2m4:5-6:5[0m

  2 [2mâ”†[0m {|
  3 [2mâ”†[0m query MyQuery {
  [1;31m4[0m [2mâ”†[0m   [1;31mlists {[0m
  [1;31m5[0m [2mâ”†[0m [1;31m    ...MissingFragment[0m
  [1;31m6[0m [2mâ”†[0m [1;31m  }[0m
  7 [2mâ”†[0m }
  8 [2mâ”†[0m |}

  [1;33mThe module or file MissingFragment can't be found.[0m
  - If it's a third-party dependency:
    - Did you list it in bsconfig.json?
    - Did you run `rescript build` instead of `rescript build -with-deps`
      (latter builds third-parties)?
  - Did you include the file's directory in bsconfig.json?

